schema/schema.sql

```diff
@@ -0,0 +1,18 @@
+-- Create Books table
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  publisher VARCHAR,
+  cover_state VARCHAR,
+  genre_id INT NULL REFERENCES genres(id),
+  author_id INT NULL REFERENCES authors(id),
+  label_id INT NULL REFERENCES labels(id),
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL
+);
```

Comment:
[OPTIONAL] Great job 👍 💯 like how you also implemented the association between the tables 🎉, I just think you forgot to include the source and it's relationship to the source table